core/io/multiplayer_api.cpp

```diff
@@ -844,6 +848,7 @@ void MultiplayerAPI::_bind_methods() {
 	ADD_SIGNAL(MethodInfo("connected_to_server"));
 	ADD_SIGNAL(MethodInfo("connection_failed"));
 	ADD_SIGNAL(MethodInfo("server_disconnected"));
+	ADD_SIGNAL(MethodInfo("rpc_node_notfound", PropertyInfo(Variant::STRING, "node_name")));
```

Comment:
Is there anything left that's required to be able to use it purely on the MultiplayerAPI itself instead of the SceneTree? (As it's the case when using CustomMultiplayer). SceneTree got a "ClassDB::bind_method" added - might that be missing?